data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/s3/S3ObjectWorker.java

```diff
@@ -64,6 +67,81 @@ public S3ObjectWorker(final S3ObjectRequest s3ObjectRequest) {
         this.s3ObjectPluginMetrics = s3ObjectRequest.getS3ObjectPluginMetrics();
     }
 
+    private void processObjectMetadata(final AcknowledgementSet acknowledgementSet,
+                               final S3ObjectReference s3ObjectReference,
+                               final BufferAccumulator<Record<Event>> bufferAccumulator,
+                               final SourceCoordinator<S3SourceProgressState> sourceCoordinator,
+                               final String partitionKey) throws IOException {
+        final S3InputFile inputFile = new S3InputFile(s3Client, s3ObjectReference, bucketOwnerProvider, s3ObjectPluginMetrics);
+        final String BUCKET = "bucket";
```

Comment:
These should all be private static fields.

Also, you can clarify the names with `_KEY`. e.g. `BUCKET_KEY`.

Reply:
I thought about it and I felt `KEY_KEY` would not be good. So, decided to not use `_KEY`